packages/amplify-graphql-relational-transformer/src/__tests__/__snapshots__/amplify-graphql-belongs-to-transformer.test.ts.snap

```diff
@@ -321,7 +321,12 @@ exports[`@belongsTo directive with RDS datasource composite key should generate
 #if( $ctx.error )
   $util.error($ctx.error.message, $ctx.error.type)
 #else
-  $util.toJson($ctx.result)
+  #set( $resultValue = $ctx.result )
+  #set( $operation = $util.defaultIfNull($ctx.source.get(\\"__operation\\"), null) )
+  #if( $operation == \\"Mutation\\" )
```

Comment:
here and other places:
Instead of checking for null and defaulting to null again, does the below simplification work?
```
#if( $ctx.source.get(\\"__operation\\") == \\"Mutation\\" )
```